presentation/dto/invitation.go

```diff
@@ -1,10 +1,27 @@
 package dto
 
-import "github.com/google/uuid"
+import (
+	. "github.com/google/uuid"
+	"split-the-bill-server/domain/model"
+)
 
-type InvitationInputDTO struct {
-	ID     uuid.UUID `json:"id"`
-	User   uuid.UUID `json:"user"`
-	Type   string    `json:"type"`
-	Accept bool      `json:"accept"`
+type GroupInvitationDTO struct {
+	Issuer  UUID   `json:"issuer"`
+	GroupID UUID   `json:"groupID"`
+	Invites []UUID `json:"invites"`
```

Comment:
There is a typo; it should be "Invitees" instead of "Invites" (please verify & correct if it appears elsewhere)